src/main/java/lotto/model/dao/RoundDAO.java

```diff
@@ -0,0 +1,76 @@
+package lotto.model.dao;
+
+import lotto.model.Money;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class RoundDAO {
+    public void updateRoundWith(String input_money) {
+        Connection connection = null;
+        PreparedStatement statement = null;
+
+        try {
+            connection = DBManager.getConnection();
+            String query = "INSERT INTO round_tb VALUES (0, ?)";
+            statement = connection.prepareStatement(query);
+
+            statement.setInt(1, Integer.parseInt(input_money));
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            System.err.println(e.getMessage());
+        } finally {
+            DBManager.close(connection, statement, null);
+        }
+    }
+
+    public int recentRound() {
+        Connection connection = null;
+        PreparedStatement statement = null;
+        ResultSet resultSet = null;
+
+        try {
+            connection = DBManager.getConnection();
+            String query = "select count(*) from round_tb";
```

Comment:
```suggestion
            String query = "select count(*) as total from round_tb";
```
별칭을 주고 resultSet.getInt("total"); 하면 나오지 않나요?

Reply:
생각치못했습니다, 알려주셔서 감사합니다.
논외로, 전반적으로 DB를 잘 모르다보니 부족한점이 많습니다, 방학시간을 비롯해 좀더 DB공부를 해보아야겠습니다..!